src/mono/mono/mini/mini-runtime.c

```diff
@@ -2062,7 +2064,22 @@ typedef struct
 	// Null terminated function name
 	// Native code
 } JitCodeLoadRecord;
+typedef struct
+{
+        guint64 code_addr;
+        guint32 line;
+        guint32 discrim;
+        char name[];
+}DebugEntry;
+typedef struct
+{
+        RecordHeader header;
+        guint64 code_addr;
+        guint64 nr_entry;
+        DebugEntry debug_entry[];
+}JitCodeDebug;
 
+static void add_basic_JitCodeDebug_info(JitCodeDebug *record);
```

Comment:
```suggestion
static void add_basic_JitCodeDebug_info (JitCodeDebug *record);
```

Reply:
In mono code base, we add a space between the function name and the "(". Please follow this function call style for all the code change in this PR.